clients/fides-js/src/components/notices/NoticeToggles.tsx

```diff
@@ -0,0 +1,57 @@
+import { h } from "preact";
+
+import { ConsentMechanism, PrivacyNotice } from "../../lib/consent-types";
+
+import Divider from "../Divider";
+
+import { GpcBadgeForNotice } from "../GpcBadge";
+import DataUseToggle from "../DataUseToggle";
+
+const NoticeToggles = ({
```

Comment:
Like `NoticeOverlay`, this is a notice specific version of a list of an existing component (`NoticeToggle` --> `DataUseToggles`)